ts-rs/Cargo.toml

```diff
@@ -12,14 +12,18 @@ categories = ["development-tools::ffi", "development-tools::build-utils", "wasm"
 readme = "../README.md"
 
 [features]
+# If enabled, each derived type will be exported in an automaticaly generated test.
```

Comment:
This is not entirely correct, since tests are only generated when `#[ts(export)]` is present. I think we should highlight in the comment that this is meant for toggling this on and off